### ISSUE TITLE
End-to-end testing

### ISSUE THREAD

**Issue by linusmarco** [fernando-mc/serverless-finch#35]
(split out from #29)

Copied from #29, @fernando-mc's current testing process is below. Let's try to come up with a more standardized and automated set of e2e tests so that multiple people (or even just our CI) can share the burden. Anyone want to volunteer to take this on? I'm happy to chip in here as well.

1. Review the code with no tests for syntax/potential errors without running it
2. Wait for comments and updates from the contributor
3. When I'm satisfied with changes I pull down the pr code from github with `pr 10` for example to get pr 10. pr is aliased to this:
```bash
alias pr='f() { git fetch origin && git checkout pr/"$1"; }; f'
```
4. I build an npm package from the modified code with `npm pack`. This outputs a gzipped(?) file.
5. I move the gzipped file to a test directory and install it with `npm install serverless-finch-long-name-withsomenumbers.gzip`
6. Then in that same directory I have a sample `serverless.yml` file and multiple subdirectories with different index.html and error.html files.
7. I run a series of `sls deploy` `sls remove` `sls deploy --region us-west-2` commands to try and emulate potential use cases.
8. I test that each command deploys/removes correctly.

**Comment by fernando-mc** [comment]
I also have a series of scripts I run now with a set of configuration files and bash commands to test functionality manually. I don't think this process is super feasible for other folks though so I'm only going to do it for tagged releases.

**Comment by mikejpeters** [comment]
Resolved by #144
